quantum/process_keycode/process_autocorrect.c

```diff
@@ -39,28 +46,36 @@ bool process_autocorrection(uint16_t keycode, keyrecord_t* record) {
         case KC_LSFT:
```

Comment:
Yep, I do have home row mods, as well as combos, bilateral combinations, and miscellaneous mod tap keys, if that makes a difference.

Reply:
Only issue with that is that it then misfires for mod tap and layer tap keys. 

However, been considering pulling out most of the check into a separate function, so that it can be customized.